params/default.go

```diff
@@ -14,6 +15,51 @@ func DefaultNetwork() Network {
 }
 
 func init() {
+	// check if custom network option set
+	if customNet, ok := os.LookupEnv("CELESTIA_CUSTOM_GENESIS"); ok {
+		fmt.Print("\n\nWARNING: Celestia custom network specified. Only use this option if the node is " +
+			"freshly created and initialized.\n**DO NOT** run a custom network over an already-existing node " +
+			"store!\n\n")
+
+		params := strings.Split(customNet, "=")
+		// ensure both params are present
+		if len(params) != 2 {
+			panic("must provide CELESTIA_CUSTOM_GENESIS in this format: <network_ID>=<genesis hash>")
```

Comment:
Nit, but would `:` be a better separator?

Reply:
We could use that too, I don't mind.

Reply:
It's mostly that `CELESTIA_CUSTOM_GENESIS=networkname=hash` looks weird, while `CELESTIA_CUSTOM_GENESIS=networkname:hash` is better IMO.

Reply:
https://github.com/celestiaorg/celestia-node/pull/717/commits/c2ac8d229a329b9d6ef592cadc7e4315f55dd059 ?